index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Mobile First</title>
+  </head>
+
+  <body>
+    <!-- Header-section starts here -->
+ <div class="hero-section">
+
+    <header>
+      <div class="container">
+       
+        <div class="mobile-status">
+          <div class="left-side">
+            <h4>9:14</h4>
+          </div>
+          <div class="right-side">
+              <img src="/images/status-icons.svg" alt="status icons" />
+            </div>
+          </div>
```

Comment:
- Please, let's remove this section as the design was been viewed in a mobile device, that why you see the time on the design. This is not meant to be in the design. :pray: